py/selenium/webdriver/common/log.py

```diff
@@ -43,7 +42,6 @@ class Log():
     """
 
     def __init__(self, driver, bidi_session) -> None:
-        assert sys.version_info >= (3, 7)
```

Comment:
Given the core library won't work with `3.6` or below fundamentally now; this looks fine to me